### ISSUE TITLE
Need Edits to GFC Docs

### ISSUE THREAD

**Issue by k-rokk** [NetAppDocs/bluexp-edge-caching#24]
### Page URL

https://docs.netapp.com/us-en/occm38/concept_gfc.html

### Page title

Learn about Global File Cache

### Summary

Can you please add info around requirements for GFC VM's running on Google GCE?  Thanks

All installations also require that you deploy two or more virtual machines (VMs) in the cloud. This results in charges from the selected cloud provider.

Global File Cache Management Server:

In Azure, this runs on a D2s_V3 or equivalent (2 vCPU/8GB RAM) VM with 127GB premium SSD

In AWS, this runs on a m4.large or equivalent (2 vCPU/8GB RAM) instance with 127GB general purpose SSD

Global File Cache Core:

In Azure, this runs on a D4s_V3 or equivalent (4 vCPU/16GB RAM) VM with 127GB premium SSD

In AWS, this runs on a m4.xlarge or equivalent (4 vCPU/16GB RAM) instance with 127GB general purpose SSD

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-tonacki** [comment]
@k-rokk, That's a good catch.  I had asked for that info when we added this support in April and did not realize I had not gotten a response.  I'll reach out again to get the answer to update the docs.

Thanks for letting us know.

Tom

**Comment by netapp-tonacki** [comment]
Hi @k-rokk, I have the details for the VM for the Core: N2-standard-4 or equivalent; Minimum: 4 vCPU/16GB RAM
(https://cloud.google.com/compute/docs/general-purpose-machines#n2_series), but I am still waiting for the info for the Management Server.

Tom

**Comment by netapp-tonacki** [comment]
Hi @k-rokk, I have added the details for the Core VM and Management VM specs for Google Cloud now.  https://docs.netapp.com/us-en/bluexp-edge-caching/concept-gfc.html#cost   There was a change to the Core VM specs for Azure and AWS at the same time, so I've updated that as well.  The Core VM now recommends 8 vCPUs and 32 GB RAM instead of 4 vCPU/16 GB RAM...

Thanks again for bringing this up.

Tom
